db/seeds.rb

```diff
@@ -670,6 +685,21 @@ def cluster(project_media, feed, team_id)
     }
     Cluster.create!(cluster_params)
   end
+
+  def imported_fact_check_params(media_type)
+    unless media_type == 'Blank' then false end
+      {
+        summary: Faker::Company.catch_phrase,
+        title: Faker::Company.name,
+        user: bot,
+        language: 'en',
+        url: get_url_for_some_fact_checks(4)
+      }
+  end
+
+  def channel(media_type)
+    media_type == "Blank" ? { main: CheckChannels::ChannelCodes::FETCH } : { main: 0}
```

Comment:
Would be nicer to replace `0` by `CheckChannels::ChannelCodes::MANUAL`, for readability.

Reply:
Agreed, will update it.